ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/DivulgedContract.scala

```diff
@@ -16,5 +16,5 @@ import com.daml.lf.value.Value
   */
 final case class DivulgedContract(
     contractId: Value.ContractId,
-    contractInst: ContractInst,
+    contractInst: Value.ContractInst[Value.VersionedValue[Value.ContractId]],
```

Comment:
We may want to move the alias for `ContractInst` to the lib where these types are being defined.

Reply:
We discussed this elsewhere. The maintainers of LF feel that using the alias is a choice and they don't encourage it or discourage it, so it should be declared as and when it's needed.